CHANGELOG.md

```diff
@@ -2,17 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-# [1.1.0-beta.0](https://github.com/segmentio/analytics-react-native/compare/v1.0.1-beta.0...v1.1.0-beta.0) (2019-06-20)
+<a name="1.1.0-beta.1"></a>
```

Comment:
I think this should be adding a new release to the changelog (rather than modifying the an already released one).